src/Components/DropdownItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  person: Person
+  onPersonSelect: (person: Person) => void;
+};
+
+export const DropdownItem: React.FC<Props> = ({ person, onPersonSelect }) => (
+  <div
+    className="dropdown-item"
+    role="button"
+    tabIndex={0}
+    onClick={() => onPersonSelect(person)}
+    onKeyDown={(e) => {
+      if (e.key === 'Enter' || e.key === 'Space') {
+        onPersonSelect(person);
+      }
+    }}
```

Comment:
Hint for the future (you may leave it as it is now) - you may use clickable elements like `button`, `option`, or `a`s - just style them appropriately